website/docs/gitops-templates/templates.mdx

```diff
@@ -240,10 +240,30 @@ spec:
         name: "{{ .params.CLUSTER_NAME }}"
 ```
 
-### Parameter metadata - `spec.params`
+
+## Parameters
 
 You can provide additional metadata about the parameters to the templates in the `spec.params` section.
 
+### Required Parameters
+Currently templates use the `CLUSTER_NAME` to determine the path in gitrepo and `RESOURCE_NAME` for the file name when we create the PR and write to your git repository. 
+
+You must provide either:
```

Comment:
implies mutually exclusive, is this the case?

Reply:
not exclusive. Both are used with a slightly different purpose.

Reply:
rephrased, changes soon will make this much simpler and more explicit. (path feature)